topology/probes/istio/destinationrule.go

```diff
@@ -48,7 +51,16 @@ func newDestinationRuleProbe(client interface{}, g *graph.Graph) k8s.Subprobe {
 }
 
 type destinationRuleSpec struct {
-	App string `mapstructure:"host"`
+	App            string      `mapstructure:"host"`
```

Comment:
Expecting field name "Host"

Reply:
As destinationRuleSpec is not really necessary here, this part is removed for now